packages/react-admin-stories/src/react-admin-core/FormCustomButtonsContainer.tsx

```diff
@@ -0,0 +1,105 @@
+import { ApolloProvider } from "@apollo/react-hooks";
+import { Button } from "@material-ui/core";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import { BeachAccess as BeachAccessIcon } from "@material-ui/icons";
+import { storiesOf } from "@storybook/react";
+import { FinalForm, IStackApi } from "@vivid-planet/react-admin-core";
+import { Field, FormPaper, Input } from "@vivid-planet/react-admin-form";
+import { styled } from "@vivid-planet/react-admin-mui";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import ApolloClient from "apollo-client";
+import { ApolloLink } from "apollo-link";
+import { RestLink } from "apollo-link-rest";
+import * as React from "react";
+import { FormRenderProps } from "react-final-form";
+
+interface IComponentProps {
```

Comment:
Das Interface ist eigentlich in der library auch definiert. `IButtonsContainerProps ` Kann man das in der library exportieren und hier importieren ?